index.html

```diff
@@ -870,6 +870,20 @@ <h3>Supporting algorithms</h3>
       </ul>
       Run the [=reset observation window=] steps and start a timer to re-run the steps when the observer.{{PressureObserver/[[ObservationWindow]]}}
       time has passed, using different randomized values.
+    <aside class="note">
+      Implementers are advised to use:
+      <ul>
+        <li>
+          a range in milliseconds between 300000 and 600000 for |observer|.{{PressureObserver/[[ObservationWindow]]}}.
```

Comment:
a range between ... in milliseconds ?

Reply:
a range between 300.000 milliseconds (5 minutes) and 600.000 milliseconds (10 minutes)  for |observer|.{{PressureObserver/[[ObservationWindow]]}}.